.pylintrc

```diff
@@ -75,6 +75,7 @@ enable=
 # --disable=W"
 disable=
     cyclic-import,  # re-enable once this no longer raises false positives
+    no-member,  # re-enable once this no longer raises false positives
```

Comment:
I mentioned the my concerns about blanket disabling of messages in the past. Is this a necessary requirement?

Reply:
Yes, it's necessary requirement before we update min supported python to 3.11

Reply:
Maybe we could add a note in the comment that this will become redundant after the min required version is 3.11

Reply:
meanwhile raised the issue on pylint GitHub https://github.com/pylint-dev/pylint/issues/8897

Reply:
@villebro Added the note in the comment